README.md

```diff
@@ -26,64 +26,62 @@ dependencies {
     implementation 'me.dmdev.rxpm:rxpm:$latest_version'
     
     // RxBinding (optional)
-    implementation 'com.jakewharton.rxbinding2:rxbinding-kotlin:$latest_version'
+    implementation 'com.jakewharton.rxbinding3:rxbinding:$latest_version'
     
-    // Conductor (if you use it)
-    implementation 'com.bluelinelabs:conductor:$latest_version'
 }
 ```
 
 ### Create a Presentation Model class and define reactive properties
 ```kotlin
-class DataPresentationModel(
-    private val dataModel: DataModel
-) : PresentationModel() {
-
-    val data = State<List<Item>>(emptyList())
-    val inProgress = State(false)
-    val errorMessage = Command<String>()
-    val refreshAction = Action<Unit>()
-
-    override fun onCreate() {
-        super.onCreate()
-
-        refreshAction.observable
-            .skipWhileInProgress(inProgress.observable)
-            .flatMapSingle {
-                dataModel.loadData()
-                    .subscribeOn(Schedulers.io())
-                    .bindProgress(inProgress.consumer)
-                    .doOnError { 
-                        errorMessage.consumer.accept("Loading data error")
-                    }
-            }
-            .retry()
-            .subscribe(data.consumer)
-            .untilDestroy()
-
-        refreshAction.consumer.accept(Unit) // first loading on create
+class CounterPm : PresentationModel() {
+
+    companion object {
+        const val MAX_COUNT = 10
+    }
+
+    val count = state(initialValue = 0)
+
+    val minusButtonEnabled = state {
+        count.observable.map { it > 0 }
+    }
+
+    val plusButtonEnabled = state {
+        count.observable.map { it < MAX_COUNT }
+    }
+
+    val minusButtonClicks = action<Unit> {
+        this.filter { count.value > 0 }
+            .map { count.value - 1 }
+            .doOnNext(count.consumer)
+    }
+
+    val plusButtonClicks = action<Unit> {
+        this.filter { count.value < MAX_COUNT }
+            .map { count.value + 1 }
+            .doOnNext(count.consumer)
     }
 }
 ```
+In this sample initialisation of the states and actions done in their blocks, but you also can do it in onCreate() or other callbacks. Don't forget to use untilDestroy() or other similar extension.
```

Comment:
In this sample the initialisation of states and actions is done in their own blocks, but it's also possible to do it in onCreate() or other callbacks. Don't forget to use untilDestroy() or other similar extension.